docs/reference/filters.md

```diff
@@ -634,6 +634,32 @@ It logs with INFO level and a unique ID per request:
 The absorbSilent filter reads and discards the payload of the incoming requests. It only
 logs read errors other than EOF.
 
+## uniformJitter
+
+The uniformJitter filter creates a jitter latency in a uniform
+distribution for requests. The first parameter is the mean and the
+second is the window. In the example we would sleep for 100ms+/-10ms.
```

Comment:
I think users would tend to pick even values for "window". 
Would it be better if we call it "delta", something like:

> The `uniformJitter` filter creates uniformly distributed request latency on the interval `[mean-delta, mean+delta]`
> * -> uniformJitter("100ms, "10ms") -> "https://www.example.org";

so that user does not have to divide "window" to calculate max or min latency.

An alternative would be to specify min and max latency directly:
> The `uniformJitter` filter creates request latency uniformly distributed between min and max.
> * -> uniformJitter("90ms, "110ms") -> "https://www.example.org";

Reply:
yeah true maybe better for the user. It was so nice in code :-D

Reply:
I changed as you suggested